specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2021-06-01-preview/examples/BackupInstanceOperations/ValidateForBackup.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
1) updated all instances of version 2021-02-01-preview to 2021-06-01-preview in file.

2) Added datasourceAuthCredentials @ line number 30